partiql-parser/src/scanner.rs

```diff
@@ -589,6 +589,11 @@ mod test {
 
     #[rstest]
     #[case::bad_identifier("💩")]
+    #[case::unterminated_line_comment("-- DROP")]
+    #[case::unbalanced_block_comment("/*\n\n SELECT /* WHERE */")]
+    #[case::unbalanced_block_comment("/* CASE do WHEN re THEN mi ELSE fa END /*")]
+    #[case::unbalanced_block_comment("/*SELECT /* FROM /* FULL OUTER JOIN */ */ ")]
+    #[case::unbalanced_block_comment("/*/*/*/*/*/*/*/*[ascii art here]*/*/*/*/*/*/*/ ")]
```

Comment:
Minor edits to the test case names:

```suggestion
    #[case::unbalanced_block_comment_nested("/*\n\n SELECT /* WHERE */")]
    #[case::unbalanced_block_comment_end_dangling("/* CASE do WHEN re THEN mi ELSE fa END /*")]
    #[case::unbalanced_block_comment_nested_open2("/*SELECT /* FROM /* FULL OUTER JOIN */ */ ")]
    #[case::unbalanced_block_comment_deeply_nested("/*/*/*/*/*/*/*/*[ascii art here]*/*/*/*/*/*/*/ ")]
```

Reply:
Reasonable naming to me, thanks.

Reply:
Thanks--I neglected to apply this suggestion, so I opened this change in #27.